spp_scanner_api/controllers/scanner_controller.py

```
@@ -0,0 +1,60 @@
+import logging
+
+from odoo import http
+from odoo.http import Response, request
+
+_logger = logging.getLogger(__name__)
+
+
+class SPPScannerController(http.Controller):
+    @http.route("/auth", type="json", auth="public", methods=["POST"], website=False)
+    def login(self, **post):
+        username = post.get("username", False)
+        password = post.get("password", False)
```

If any is missing we should return a validation error.